CONTRIBUTING.md

```diff
@@ -27,57 +29,36 @@ Here's an example of what your `index.md` might look like:
 ```js
 ---
 {
-  // "name" should be your displayed name, however you want it
-  // to appear on your posts
```

Comment:
Let's not remove these comments. They add quick glance-ability to the page when someone's glancing at the docs.